backend/src/v5/services/swagger/swagger.schemas.js

```diff
@@ -59,6 +59,7 @@ const ticketTemplatePropSchema = {
 			name: helpers.stringDef('Name of the prop', 'Floor'),
 			type: helpers.stringDef('Property type', propTypes.ONE_OF, Object.values(propTypes)),
 			deprecated: helpers.boolDef('Denotes if this prop is no longer in use', false),
+			immutable: helpers.boolDef('Denotes if this prop can be altered once it is set', false),
```

Comment:
The description is misleading, as it's implying if it's true it can be altered. Immutable is a valid word in the dictionary, so let's just use that.
```suggestion
			immutable: helpers.boolDef('Denotes if this prop is immutable', false),
```